tests/cli/test_report.py

```diff
@@ -55,6 +55,11 @@ def test_report():
     result.print_report()
 
 
+def test_report_detailed():
+    # buildtest report --detailed
+    Report(configuration=configuration, format_detailed=True)
```

Comment:
please add another test to catch exception when `--detailed` and `--format` is specified when invoking `Report` class